core/block.ts

```diff
@@ -761,8 +759,6 @@ export class Block implements IASTNodeLocation, IDeletable {
     const blocks = [this];
     const childBlocks = this.getChildren(ordered);
     for (let child, i = 0; child = childBlocks[i]; i++) {
-      // AnyDuringMigration because:  Argument of type 'Block[]' is not
-      // assignable to parameter of type 'this[]'.
       blocks.push(...child.getDescendants(ordered) as AnyDuringMigration);
```

Comment:
Did you mean to remove the cast here as well?

Reply:
I meant to leave the comment in because the cast was not trivially resolved. Fixed.